components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -5325,7 +5325,7 @@ public void updatePermissionListOfGroup(String groupName, String[] permissionToA
         return filteredFlatAttributeMap;
     }
 
-    private Map<String, Attribute> getFilteredEnterpriseUserSchemaAttributes(Map<ExternalClaim, LocalClaim>
+    private Map<String, Attribute> getFilteredEnterpriseUserAndCustomSchemaAttributes(Map<ExternalClaim, LocalClaim>
```

Comment:
```suggestion
    private Map<String, Attribute> getFilteredSchemaAttributes(Map<ExternalClaim, LocalClaim>
```

Can you add a method level comment saying it will filter supportedByDefault attributes and return?

Reply:
Fixed in commit 68ac3db52b0b27c9b4562c9d31eafebd87e8061a